nautobot/dcim/templates/dcim/controllermanageddevicegroup_create.html

```diff
@@ -0,0 +1,88 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+{% load static %}
+
+{% block form %}
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Controller</strong></div>
+        <div class="panel-body">
+            {% render_field form.controller.as_hidden %}
+            {% render_field form.controller %}
+            {% render_field form.name %}
+            {% render_field form.devices %}
```

Comment:
Can you test and confirm that this form works correctly both when creating a new group with devices and also when editing an existing group that already has some devices?

Reply:
Tested that and works properly in both cases.